### PR TITLE
Feature:  Implented Square Devel

### PR DESCRIPTION
**Pull request by mo3pheus**
Square class implements IsShape interface.

### CODE REVIEW

File: src/main/java/bootstrap/Square/Square.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package bootstrap.Square;
+
+import bootstrap.IsShape;
+
+public class Square implements IsShape {
+
+    int sideLength;
+
+    public Square(int sideLength) {
```
Comment (mo3pheus) [inline src/main/java/bootstrap/Square/Square.java]: Please change the datatype to a double.

File: src/main/java/bootstrap/Square/Square.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package bootstrap.Square;
+
+import bootstrap.IsShape;
+
+public class Square implements IsShape {
```
Comment (mo3pheus) [inline src/main/java/bootstrap/Square/Square.java:5]: Write a test case, showing double. as well as integer inputs for Square.java

**Review by mo3pheus — COMMENTED** [review]
Address review comments.

File: src/main/java/bootstrap/Square/Square.java (hunk 1 above)
Comment (mo3pheus) [reply src/main/java/bootstrap/Square/Square.java]: Addressed as requested.

File: src/main/java/bootstrap/Square/Square.java (hunk 2 above)
Comment (mo3pheus) [reply src/main/java/bootstrap/Square/Square.java:5]: addressed comment

**Review by sahaya-cyril — APPROVED** [review]
The code meets coding standards

**Review by AnthonyThomas20 — APPROVED** [review]
The code meets the coding standards.

**Pull request merged**